src/serialization/serialization.py

```diff
@@ -0,0 +1,29 @@
+import pickle
```

Comment:
You should really use an `__init__.py` but in the interest of time I won't insist on it.

Reply:
cc @ostapstephan

Reply:
Closed.